test/methods.sh

```diff
@@ -39,12 +39,44 @@ describe '_stub::methods'
   end # returns
 
   describe 'errors'
+    # before
+      stub 'bad'
+    # end
```

Comment:
Yes, shpec is awful. I'm going to be rewriting it as a weekend project soon to add real testing constructs.